src/ripple/net/impl/RPCCall.cpp

```diff
@@ -1331,6 +1387,13 @@ rpcClient(std::vector<std::string> const& args,
         nRet                    = rpcINTERNAL;
     }
 
+    if (jvRequest.isMember("jsonrpc"))
+    {
+        jvOutput["jsonrpc"] = jvRequest["jsonrpc"];
```

Comment:
Micro-nit: we have this same logic in at least three different places. Maybe having a single utility function that we call makes sense?